File: reporters/test/artifacts/dotnet/mstest-failing/CalculatorTests.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace Calculator.Tests
{
    [TestClass]
    public class CalculatorTests
    {
        [TestMethod]
        public void Should_add_numbers_correctly()
        {
            Assert.AreEqual(6, 2 + 3);
        }
    }
}

File: reporters/test/artifacts/dotnet/mstest-vb-passing/CalculatorTests.vb
Imports Microsoft.VisualStudio.TestTools.UnitTesting

Namespace Calculator.Tests

    <TestClass>
    Public Class CalculatorTests

        <TestMethod>
        Public Sub Should_add_numbers_correctly()
            Assert.AreEqual(5, 2 + 3)
        End Sub

    End Class

End Namespace
